### ISSUE TITLE
Invalid last path element for adding node 

### ISSUE THREAD

**Issue by sansai2011** [TWCable/grabbit#176]
On behalf of #175 

Please suggest us what can be done to complete the task not to give the exception for Invalid last path element for adding node.
1. Either can we exclude such node names?
If we can please give the example.
2. or any workaround to forcefully write the same node even if not correct.

Anyway should complete the task.

Thanks in Advance.

**Comment by jbornemann** [comment]
You have two options.

1.) Rename the nodes to include valid characters.
2.) Exclude the nodes from your sync. You may choose to use excludePaths. Check out the example on the README; it shows an example of this.

**Comment by sansai2011** [comment]
In readme only the nodes at specific path(predefined) can exclude as i understand but if i dont know where at which path invalid nodes are there, in such how can i exclude?

**Comment by sansai2011** [comment]
Is it possible if create PR to forcefully write the invalid node?

**Comment by jbornemann** [comment]
No, this is not something that Grabbit can, or should handle.

**Comment by sansai2011** [comment]
its ok, but even if the exception occurs, at least job should complete the task for all other nodes. its getting stucked wherever invalid node found.should move for other process if i am not wrong.

**Comment by jbornemann** [comment]
Yes, that job path will fail in it's entirety if a node is unable to be synched. We perform this way to keep it simple, and predictable. It would be possible to sync other nodes on the same node level, but we couldn't successfully sync a failed node's children; so it really doesn't hold a lot of value to continue. 

If a node fails to sync, the best course of action is to correct why the node is un able to sync, and continue from there.
